### PR TITLE
ingest2parquet updates

### PR DESCRIPTION
**Pull request by sapthasurendran**
## Why are these changes needed?
- Removed Error messages from log for files skipped
- Added make targets to run local and s3 samples using minio 
- Associated readme changes

## Related issue number (if any).
#46 
#47 
#73

### CODE REVIEW

File: tools/ingest2parquet/.make.tools [hunk 1]
```diff
@@ -0,0 +1,23 @@
+include $(REPOROOT)/.make.defaults
+
+# Expects RUN_FILE RUN_ARGS
+.PHONY: .tools.run-src-file
```
Comment (daw3rd) [inline tools/ingest2parquet/.make.tools]: This is a cut/paste from .make.transforms.  Can you

1.  Move this to .make.defaults
2. Rename to `.defaults.run-src-file`
3. And replace the definintion of  `.transforms.run-src-file:` in .make.transforms with `.transforms.run-src-file: .defaults.run-src-file`

**Review by daw3rd — CHANGES_REQUESTED** [review]

File: transforms/.make.transforms [hunk 2]
```diff
@@ -224,7 +224,7 @@ conventions:
 
 # Expects RUN_FILE
 .PHONY: .transforms.run-src-file
-.transforms.run-src-file:
+.transforms.run-src-file: .defaults.run-src-file
```
Comment (daw3rd) [inline transforms/.make.transforms:227]: should the body of this target be removed now that it calls .defaults?

Comment (daw3rd) [reply transforms/.make.transforms:227]: otherwise, i think this will run twice.

**Review by daw3rd — CHANGES_REQUESTED** [review]

File: transforms/.make.transforms (hunk 2 above)
Comment (sapthasurendran) [reply transforms/.make.transforms:227]: yes , removed the body and tested. works fine.

**Review by daw3rd — APPROVED** [review]

File: transforms/.make.transforms (hunk 2 above)
Comment (daw3rd) [reply transforms/.make.transforms:227]: please mark associated issues fixed.

**Comment by daw3rd** [comment]
Please mark associated issues as fixed.

**Pull request merged**